### PR TITLE
[PW-1700] Add the terminal configuration fields in the back office

### PR DESCRIPTION
**Pull request by AlexandrosMor**
**Description**

**Tested scenarios**

**Fixed issue**:

### CODE REVIEW

File: adyenv6core/resources/localization/adyenv6core-locales_en.properties [hunk 1]
```diff
@@ -40,6 +40,12 @@ type.basestore.adyenRecurringContractMode.name=Recurring contract type
 type.basestore.adyenRecurringContractMode.description=How the payment will be stored on Adyen
 type.basestore.adyenBoleto.name=Enable Boleto
 type.basestore.adyenBoleto.description=Enables Boleto as an available payment method when delivery country is Brasil and currency is BRL. Requires Web Service User configuration.
+type.basestore.adyenPosEnabled.name=Enable Pos
```
Comment (rkewlani) [inline adyenv6core/resources/localization/adyenv6core-locales_en.properties]: Fir display name, Pos should be POS (all caps).

File: adyenv6core/resources/localization/adyenv6core-locales_en.properties [hunk 2]
```diff
@@ -40,6 +40,12 @@ type.basestore.adyenRecurringContractMode.name=Recurring contract type
...
+type.basestore.adyenPosEnabled.description=Enable Pos as an available payment method
+type.basestore.adyenPosMerchantAccount.name=Pos merchant account
+type.basestore.adyenPosApiKey.name=Pos API key
```
Comment (rkewlani) [inline adyenv6core/resources/localization/adyenv6core-locales_en.properties]: API Key for POS merchant

File: adyenv6backoffice/resources/adyenv6backoffice-backoffice-labels/labels_en.properties [hunk 3]
```diff
@@ -23,3 +23,4 @@ hmc.adyen.notifications=Server Notification Settings
 hmc.adyen.api=API Settings
 hmc.adyen.hpp=Hosted Payment Pages Settings
 hmc.adyen.advanced=Advanced Settings
+hmc.adyen.terminalApi=Terminal API settings
```
Comment (rkewlani) [inline adyenv6backoffice/resources/adyenv6backoffice-backoffice-labels/labels_en.properties]: We should keep it pos settings. We won't use terminal Api anywhere.

File: adyenv6core/resources/adyenv6core-items.xml [hunk 4]
```diff
@@ -202,6 +202,34 @@
                     <modifiers optional="false"/>
                     <defaultvalue>Boolean.FALSE</defaultvalue>
                 </attribute>
+                <attribute qualifier="adyenPOSEnabled" type="java.lang.boolean">
```
Comment (rkewlani) [inline adyenv6core/resources/adyenv6core-items.xml]: Same in this file. Terminal API should not be anywhere.

Comment (AlexandrosMor) [reply adyenv6core/resources/adyenv6core-items.xml]: @rkewlani Can you review again ? thanks

File: adyenv6backoffice/resources/adyenv6backoffice-backoffice-config.xml [hunk 5]
```diff
@@ -32,18 +32,26 @@
...
-				</editorArea:section>
 				<editorArea:section name="hmc.adyen.notifications">
 					<editorArea:attribute qualifier="adyenNotificationUsername"/>
 					<editorArea:attribute qualifier="adyenNotificationPassword" editor="com.hybris.cockpitng.editor.defaultpassword"/>
 				</editorArea:section>
+				<editorArea:section name="hmc.adyen.pos">
+					<editorArea:attribute qualifier="adyenPOSEnabled"/>
+					<editorArea:attribute qualifier="adyenPOSApiKey" editor="com.hybris.cockpitng.editor.defaultpassword"/>
+					<editorArea:attribute qualifier="adyenPOSMerchantAccount"/>
+					<editorArea:attribute qualifier="adyenPOSStoreID"/>
```
Comment (rkewlani) [inline adyenv6backoffice/resources/adyenv6backoffice-backoffice-config.xml]: still showing ID

File: adyenv6core/resources/adyenv6core-items.xml [hunk 6]
```diff
@@ -202,6 +202,34 @@
...
+                    <description>POS Merchant Account</description>
+                    <persistence type="property"/>
+                    <modifiers optional="true"/>
+                </attribute>
+                <attribute qualifier="adyenPOSApiKey" type="java.lang.String">
+                    <description>POS API Key</description>
+                    <persistence type="property"/>
+                    <modifiers encrypted="true" optional="true"/>
+                </attribute>
+                <attribute qualifier="adyenPOSStoreId" type="java.lang.String">
```
Comment (rkewlani) [inline adyenv6core/resources/adyenv6core-items.xml]: It is Id here

File: adyenv6core/resources/adyenv6core-items.xml [hunk 7]
```diff
@@ -202,6 +202,34 @@
...
+                    <description>POS Store ID</description>
+                    <persistence type="property"/>
+                    <modifiers optional="true"/>
+                </attribute>
+                <attribute type="RecurringContractMode" qualifier="adyenPOSRecurringContractMode"
+                           generate="true" autocreate="true">
+                    <persistence type="property"/>
+                    <modifiers optional="false"/>
+                    <defaultvalue>em().getEnumerationValue("RecurringContractMode","ONECLICK")</defaultvalue>
```
Comment (rkewlani) [inline adyenv6core/resources/adyenv6core-items.xml]: It should be NONE or ONECLICK ?

File: adyenv6core/resources/adyenv6core-items.xml (hunk 6 above)
Comment (AlexandrosMor) [reply adyenv6core/resources/adyenv6core-items.xml]: I see the adyenPOSStoreId for qualifier here. Do you mean the description ?<description>POS Store ID</description> to <description>POS Store Id</description>

Comment (rkewlani) [reply adyenv6core/resources/adyenv6core-items.xml]: Yes I mean it is not consistent. So this is correct one.(adyenPOSStoreId)

Comment (AlexandrosMor) [reply adyenv6core/resources/adyenv6core-items.xml]: The qualifier is adyenPOSStoreId so it is correct.

**Pull request merged**